<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>134</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PALETAS DE CORES</string>
    </property>
   </widget>
   <widget class="Plotter" name="widgetTela" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>551</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>171</width>
      <height>104</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_G">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider_B">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcdNumber_5">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcdNumber_6">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>r</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcdNumber_4">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>b</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>g</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_R">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>135</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Raio</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_Raio">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_10">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Plano</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_Planos">
           <property name="maximum">
            <number>40</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_11">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>440</y>
      <width>551</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_X">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_Y">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_Z">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLCDNumber" name="lcdNumber_3">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_2">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="AlterarCor">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOFF">
    <property name="title">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuNovo_Desenho">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo_Desenho"/>
    <addaction name="actionAbrir_Aquivo"/>
   </widget>
   <addaction name="menuOFF"/>
   <addaction name="menuNovo_Desenho"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSalvar"/>
   <addaction name="separator"/>
   <addaction name="actionplotVoxel"/>
   <addaction name="separator"/>
   <addaction name="actioncutVoxel"/>
   <addaction name="separator"/>
   <addaction name="actionplotBox"/>
   <addaction name="separator"/>
   <addaction name="actioncutBox"/>
   <addaction name="separator"/>
   <addaction name="actionplotEsfera"/>
   <addaction name="separator"/>
   <addaction name="actioncutEsfera"/>
   <addaction name="separator"/>
   <addaction name="actionplotElipse"/>
   <addaction name="separator"/>
   <addaction name="actioncutElipse"/>
  </widget>
  <action name="actionNovo_Desenho">
   <property name="text">
    <string>Novo Desenho</string>
   </property>
  </action>
  <action name="actionAbrir_Aquivo">
   <property name="text">
    <string>Abrir Aquivo</string>
   </property>
  </action>
  <action name="actionplotVoxel">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/plotVox.svg</normaloff>:/icones/recursos/plotVox.svg</iconset>
   </property>
   <property name="text">
    <string>plotVoxel</string>
   </property>
  </action>
  <action name="actioncutVoxel">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/cortVox.svg</normaloff>:/icones/recursos/cortVox.svg</iconset>
   </property>
   <property name="text">
    <string>cutVoxel</string>
   </property>
  </action>
  <action name="actionplotBox">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/plotBox.svg</normaloff>:/icones/recursos/plotBox.svg</iconset>
   </property>
   <property name="text">
    <string>plotBox</string>
   </property>
  </action>
  <action name="actioncutBox">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/cortBox.svg</normaloff>:/icones/recursos/cortBox.svg</iconset>
   </property>
   <property name="text">
    <string>cutBox</string>
   </property>
  </action>
  <action name="actionplotEsfera">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/plotCirc.svg</normaloff>:/icones/recursos/plotCirc.svg</iconset>
   </property>
   <property name="text">
    <string>plotEsfera</string>
   </property>
  </action>
  <action name="actioncutEsfera">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/cortCirc.svg</normaloff>:/icones/recursos/cortCirc.svg</iconset>
   </property>
   <property name="text">
    <string>cutEsfera</string>
   </property>
  </action>
  <action name="actionplotElipse">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/plotElip.svg</normaloff>:/icones/recursos/plotElip.svg</iconset>
   </property>
   <property name="text">
    <string>plotElipse</string>
   </property>
  </action>
  <action name="actioncutElipse">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/cortElip.svg</normaloff>:/icones/recursos/cortElip.svg</iconset>
   </property>
   <property name="text">
    <string>cutElipse</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/salvar.png</normaloff>:/icones/recursos/salvar.png</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
   <slots>
    <slot>setVermelho(int)</slot>
    <slot>setVerde(int)</slot>
    <slot>setAzul(int)</slot>
    <slot>setTransp(int)</slot>
    <slot>alteraCor()</slot>
    <slot>mudaB(int)</slot>
    <slot>mudaG(int)</slot>
    <slot>salvaEscultor()</slot>
    <slot>executaGeomview()</slot>
    <slot>limpaEscultor()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider_X</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Z</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_R</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_B</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_G</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Raio</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_10</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>774</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Planos</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_11</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>752</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
